### PR TITLE
Thank you note to front end instructors complete.

### PR DESCRIPTION
**Pull request by ghost**
Received 9 thumbs up in slack by fellow classmates thus far

### CODE REVIEW

**Comment by evangdesigns** [comment]
1) Well said.
2) Merge the latest master with your branch; You're a commit behind.
3) Then 👍

**Comment by ghost** [comment]
merged in most recent master

File: src/index.html [hunk 1]
```diff
@@ -98,6 +98,9 @@ <h1>TECHNOLOGIES</h1>
     <h1>THANK YOU</h1>
     <p>YOU be the shit.</p>
     <h2>To Zoe, Greg, and Michael</h2>
+    <p>Thank you for helping us establish a solid foundation in the world of Software Development. Through all the Git conflicts, pits of despair, endless questions, screen share sessions, pinterest projects, pies, laughs, and everything in between, we are forever grateful for your time and concerted efforts to pass on your knowledge. You were instrumental in our growth over the past year, and without the fundamentals and patience you taught us, we would not be where we stand today as developers.  </p>
+    <h2>To Nathan, Martin, and Jameka</h2>
+    <p>Thanks persons two</p>
     <p>Thanks persons one</p>
```
Comment (aclai4067) [inline src/index.html]: this placeholder line needs to be removed please!

Comment (ghost) [reply src/index.html]: done. Thanks!

**Comment by evangdesigns** [comment]
👍

**Pull request merged**